clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -168,15 +168,26 @@ describe("System management with Plus features", () => {
       cy.getByTestId("input-name").should("not.be.disabled");
     });
 
-    it("allows changes to data uses for non-GVL vendors", () => {
+    it("don't allow editing declaration name after creation", () => {
       cy.getSelectValueContainer("input-name").type("L{enter}");
       cy.getByTestId("save-btn").click();
       cy.wait(["@postSystem", "@getSystem", "@getSystems"]);
       cy.getByTestId("tab-Data uses").click();
       cy.getByTestId("add-btn");
       cy.getByTestId("delete-btn");
       cy.getByTestId("row-functional.service.improve").click();
-      cy.getByTestId("input-name").should("not.be.disabled");
+      cy.getByTestId("input-name").should("be.disabled");
```

Comment:
I think it's better to leave this test relatively in-place so we're still checking to make sure other fields on the data use can be edited in this scenario.  You can add the check that the name and the data use inputs are disabled to the new test you added, then change this line to verify that changes are allowed to one of the other inputs instead.

Reply:
Okay. So I'll leave the original test, just chaging the final line to check for another input. And then do my new tests below.